firstpartydata/tests/resolvefpd/global-and-bidder-fpd-app-content-data-user-data.json

```diff
@@ -0,0 +1,114 @@
+{
+  "description": "Global and bidder FPD defined for app and user. Global FPD has app.content.data and user.data",
```

Comment:
This description incorrectly states that there is global and bidder specific FPD for user but there isn't any bidder specific user FPD.

Reply:
Yes, modified